Sources/ApodiniDatabase/Handlers/Read.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Fluent
+import Apodini
+
+/// A Handler that returns an array of `DatabaseModel` which all conform to the query parameters of the request. It accepts all parameters that represent properties of the specified `DatabaseModel`.
+/// It uses the database that has been specified in the `DatabaseConfiguration`.
+/// An Example using a REST request:
+/// Group("api", "birds") {
+///     Read<Bird>()
+/// }
+/// Sending a request to  ~/api/birds/birds?age=19&name=Foo would return an array of `Bird` object that have an age of 19 and the name Foo.
+public struct Read<Model: DatabaseModel>: Handler {
+    @Apodini.Environment(\.database)
+    private var database: Fluent.Database
+
+    @Properties
+    private var dynamics: [String: Apodini.Property]
+    
+    public init() {
+        var dynamicValues: [String: Parameter<String?>] = [:]
+        let infos = QueryBuilder.info(for: Model.self)
+        for info in infos {
+            dynamicValues[info.key.description] = Parameter<String?>(.http(.query))
+        }
+        _dynamics = Properties(wrappedValue: dynamicValues)
+    }
+
+    public func handle() -> String {
+//    public func handle() -> [Model] {
```

Comment:
I think the current implementation doesn't return the actual `Models` as this functionality is commented out.

Reply:
Some of the handlers currently return `String` because calling `wait()` on the `EventloopFuture` returned by the database operation results in an failed precondition as one is not allowed to call `wait()` on an `eventloop`.

```
Precondition failed: BUG DETECTED: wait() must not be called when on an EventLoop.
```

As you already suggested, it would beneficial to wait the merger of #94 or #98. If `EventloopFuture` works again, I would of course change it. So if you agree, I'd suggest we leave it like that for now.

Reply:
#98 should be merged pretty soon so I would be optimistic to have `EventLoopFuture` as possible `Handler` return types by tomorrow's team meeting 👍